### PR TITLE
simple CLI application

### LINKED ISSUE #41: CLI
Is there any plan to add a CLI [like smartcrop.js](https://github.com/jwagner/smartcrop-cli)? It would be great to be able to use smartcrop from a shell.

### PR DESCRIPTION
**Pull request by Pomyk**
closes #41

### CODE REVIEW

File: cmd/smartcrop/main.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *	Authors:
+ *		Christian Muehlhaeuser <muesli@gmail.com>
+ *		Michael Wendland <michael@michiwend.com>
+ *		Bjørn Erik Pedersen <bjorn.erik.pedersen@gmail.com>
```
Comment (muesli) [inline cmd/smartcrop/main.go]: Feel free to add yourself to the authors here!

File: cmd/smartcrop/main.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	input := flag.String("input", "", "input filename")
+	output := flag.String("output", "", "output filename")
+	w := flag.Int("width", 0, "crop width")
+	h := flag.Int("height", 0, "crop height")
+	resize := flag.Bool("resize", true, "resize after cropping")
+	quality := flag.Int("quality", 85, "jpeg quality")
+	flag.Parse()
+
+	if *input == "" {
+		fmt.Fprintln(os.Stderr, "No input file given")
```
Comment (muesli) [inline cmd/smartcrop/main.go]: Should probably return an error (`os.Exit(1)`) whenever we bail out in main.

File: cmd/smartcrop/main.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+		return
+	}
+
+	f, err := os.Open(*input)
+	defer f.Close()
```
Comment (muesli) [inline cmd/smartcrop/main.go]: Move this line below the error check. If an error occurred we must not call `f.Close()`.

**Comment by muesli** [comment]
Nice work, only a few remarks!

File: cmd/smartcrop/main.go (hunk 2 above)
Comment (Pomyk) [reply cmd/smartcrop/main.go]: done

**Pull request merged**